data/config.yml

```diff
@@ -395,7 +395,11 @@ rbac:
       selectors:
         - '[data-ouia-page-group="settings"] a[href="/settings/rbac/groups/add-group"]'
         - '[data-ouia-page-group="settings"] a[href="/settings/rbac/groups/add-group"] button'
-        - '[data-ouia-page-group="settings"] button:contains("Create groups")'
+        - '[data-ouia-page-group="settings"] button:contains("Create group")'
+    complete_group_creation:
+      selectors:
+        - '[data-ouia-page-group="settings"] a[href="/settings/rbac/groups/add-group"]button:contains("Create")
```

Comment:
Margot, I think you forgot to close the quotes here.
Thus this is failing the automated check.

Also I think the addition of that new Feature does not follow @allieh782 s standards.

Reply:
Ok I think Allie's adding it on my behalf so i'm just going to close this PR